includes/class-freemius.php

```diff
@@ -23248,7 +23279,24 @@ function _affiliation_page_render() {
 
             fs_enqueue_local_style( 'fs_affiliation', '/admin/affiliation.css' );
 
-            $vars = array( 'id' => $this->_module_id );
+            $is_bundle_context = ! is_null( $this->get_bundle_id() );
+
+            $plugin_title = $this->get_plugin_title();
+            if ( $is_bundle_context ) {
+                $plugin_title = $this->plugin_affiliate_terms->plugin_title;
+                // Add the suffix "Bundle" only if the word is not present in the title itself.
+                if ( false === mb_stripos( $plugin_title, fs_text_inline( 'Bundle', 'bundle' ) ) ) {

```

Comment:
@swashata can we assume `mb_` functions are available for all PHP setups?

Reply:
@vovafeldman The `server-environment` [handbook says](https://make.wordpress.org/hosting/handbook/server-environment/#php-extensions) that WordPress "highly recommends" `mbstring` extension. I did a little digging and it looks like [WordPress core has direct use of](https://core.trac.wordpress.org/browser/tags/5.9/src/wp-includes/functions.php#L5909) `mb_substr` with which we can assume that `mb_` functions are available for PHP setups where WordPress works. 

I used `mb_stripos` here (as you might've guessed already) to properly get the position of translated `"Bundle"` text from the plugin title. How about we keep this for now and come back at it only if it creates trouble?

Reply:
@swashata if core relies on it we can rely on it too.